test/MigrateMiddlewareToRequestHandler/TestAsset/expected/MyMiddlewareWithHandler.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
```

Comment:
We need a license docblock here and on the other test asset; we should likely also add the `declare(strict_types=1)`, though that's not completely necessary.